src/main/java/io/github/mivek/facade/MetarFacade.java

```diff
@@ -45,11 +45,8 @@ public Metar retrieveFromAirport(final String pIcao) throws IOException, ParseEx
         URL url = new URL(website);
         URLConnection urlCo = url.openConnection();
         try (BufferedReader br = new BufferedReader(new InputStreamReader(urlCo.getInputStream()))) {
-            String line = (String) br.lines().toArray()[1];
-            br.close();

```

Comment:
There's no need to manually close an `AutoCloseable` implementation that is defined in a try-with-resources block